enhancements/machine-api/aws-placement-groups.md

```diff
@@ -0,0 +1,487 @@
+
+- The following instance types are supported:
+  - Current generation instances, except for burstable performance instances (for example, T2) and Mac1 instances.
+  - The following previous generation instances: `A1`, `C3`, `cc2.8xlarge`, `cr1.8xlarge`, `G2`, `hs1.8xlarge`, `I2`,
+    and `R3`.
+- A cluster placement group can't span multiple Availability Zones.
+- You can launch multiple instance types into a cluster placement group. However, this reduces the likelihood that the
+  required capacity will be available for your launch to succeed. We recommend using the same instance type for all
+  instances in a cluster placement group.
+
+##### Spread Placement Groups
+
+A Spread placement group is a grouping of instances all placed in distinct racks within the datacenter.
+This ensures that each instance within the group has its own network and power source, maximising resillience to
+hardware failures.
+
+Spread placement groups are recommended for launching a small number of instances that must be kept separate, eg a
+cluster of etcd instances.
+
+As Spread placement groups keep instances in different racks, they can be expanded over time and can also mix instance
+types without detriment.
+
+Spread placement groups have a limited capacity. You can only launch seven instances within a Spread placement group
+within a single availability zone.
+
+###### Limitations of Spread Placement Groups
+
+- Spread placement groups are not supported for AWS Dedicated Instances.
+
+##### Partition Placement Groups
+
+A Partition placement group divides your instances into logical segments called partitions.
+Each partition is assigned a set of racks within the Availability Zone, each rack in turn having its own network and
+power source.
+No rack within the AZ will appear in more than one partition.
+
+Users may either create instances and have them spread automatically across partitions or they may allocate instances
+to a particular partition.
+Partitions are identified numerically, indexed from 1.
+
+Partition placement groups are recommended for deploying large distributed and replicated workloads across distinct
+racks.
+
+Partition placement groups are limited to seven partitions per Availability Zone.
+
+When using a Partition placement group, if users wish to control the placement of their workloads within Partitions,
+they will need to create multiple MachineSets and assign each one to a partition.
+Once they have assigned different MachineSets to partitions, they will be able to use Node affinity to assign the
+workloads to the Nodes created by the Machines within the MachineSet by selecting based on the MachineSets unique
+labels.
+
+###### Limitations of Partition Placement Groups
+
+- A partition placement group with AWS Dedicated Instances can have a maximum of two partitions.
+
+#### Creation of Placement Groups
+
+To enable users to get started with Placement Groups, we will allow users to specify the name of the placement group to
```

Comment:
how?

Reply:
Have expanded to include that they specify it via `placement.groupName`